sonar-core/src/test/java/org/sonar/core/issue/workflow/IssueWorkflowTest.java

```diff
@@ -139,7 +139,7 @@ public void fail_if_unknown_status_when_listing_transitions() {
   }
 
   @Test
-  public void do_automatic_transition() {
+  public void automatically_close_resolved_issue() {
```

Comment:
I find it odd to see only formatting changes in the unit test.
There does not seem to be a test covering closing an issue for a manual rule (which might explain why the bug existed, if I understood correctly the fix).